packages/astro/src/cli/add/index.ts

```diff
@@ -154,23 +147,28 @@ export async function add(names: string[], { flags }: AddOptions) {
 	switch (installResult) {
 		case UpdateResult.updated: {
 			if (integrations.find((integration) => integration.id === 'tailwind')) {
-				await setupIntegrationConfig({
-					root,
-					logger,
+				const dir = new URL('./styles/', new URL(userConfig.srcDir ?? './src/', root));
+				const styles = new URL('./global.css', dir);
+				if (!existsSync(styles)) {
+					logger.info(
+						'SKIP_FORMAT',
+						`\n  ${magenta(`Astro will scaffold ${green('./src/styles.global.css')}.`)}\n`,
```

Comment:
`styles.global.css`?

Probably my ignorance here but figured I would comment just in case

Reply:
Yep that's a typo, thanks!

Reply:
In that case there was a second instance of it further down 😅